projboard/tests/test_client.py

```diff
@@ -111,3 +111,27 @@ def test_aboutpage(client):
     template_names = set(tmpl.origin.template_name for tmpl in response.templates)
     # And check if that 'about/about.html' is in the set.
     assert 'about/about.html' in template_names
+
+
+@pytest.mark.django_db
+def test_get_readArticlePage(client, article):
```

Comment:
Please use python naming conventions
https://peps.python.org/pep-0008/#method-names-and-instance-variables

Reply:
done